.github/workflows/docs.yml

```
@@ -0,0 +1,25 @@
+name: docs
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Build-docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18.15
+      - name: Install dependencies
```

Let's add some cache

```suggestion
      - uses: actions/cache@v4
         id: cache
         with:
           path: '**/node_modules'
           key: npm-v3-${{ hashFiles('**/package-lock.json') }}
```